completions/rsync

```diff
@@ -1,5 +1,36 @@
 # bash completion for rsync                                -*- shell-script -*-
 
+_rsync_vercomp() {
+    if [[ $1 == $2 ]]
+    then
```

Comment:
The format doesn't seem to be the one filtered by `shfmt`. Have you set up pre-commit? It is partly described in the second last item of [CONTRIBUTING.md](https://github.com/scop/bash-completion/blob/master/CONTRIBUTING.md).

Reply:
If the description in CONTRIBUTING.md does not work, you can also check https://github.com/scop/bash-completion/pull/683#issuecomment-1010579492.

Reply:
That won't work for me, unfortunately, starting with docker probably. But I've made my best effort to install and use the tools (shfmt and shellcheck found errors).